shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1020,6 +1037,9 @@ public AccessibilityNodeInfo findFocus(int focus) {
                 if (accessibilityFocusedSemanticsNode != null) {
                     return createAccessibilityNodeInfo(accessibilityFocusedSemanticsNode.id);
                 }
+                if (embeddedFocusedNodeId != null) {
```

Comment:
What about INPUT_FOCUS for embedded views? Finding the node that has input focus is important for text input via a11y.

Reply:
added tracking for input focus as well.